### PR TITLE
Add newFeature and don't know hotfix

### PR DESCRIPTION
**Pull request by uncle9x9**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 index.html
 Swithed to github-uncle9x9 which is fetched from origin
 Switched to newFeature1 branch and added line 3 into index.html
-This is a hotfix and apply to main directly.
```
Comment (WinsonLiGeneration) [inline index.html:4]: Please create branch to do the changes and please follow pull request to do at least one review and one approval.

File: index.html [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
+This is a hotfix and apply to main directly. (Append newFeature after hotfix) I don't know the hotfix of the main and continue typing new feature in newFeatureWithHotFix branch.
```
Comment (WinsonLiGeneration) [inline index.html:4]: Conflict solved

**Review by WinsonLiGeneration — COMMENTED** [review]
@winsonliwh , conflict solved. Please approve.

**Review by winsonliwh — APPROVED** [review]
After checking, the solve conflict is ok.

**Pull request merged**